src/rf/apps/workers/process.py

```diff
@@ -58,48 +63,84 @@ def start_polling(self):
                         delete_message = self.check_timeout(record['data'])
                     elif job_type in JOB_VALIDATE:
                         if record['eventSource'] == 'aws:s3':
-                            delete_message = self.validate_image(record)
+                            delete_message = self.__validate_image(record)
 
                 if delete_message:
                     self.q.delete_message(message)
 
-    def validate_image(self, data):
+    def __validate_image(self, data):
```

Comment:
I think these steps are becoming complex enough that they could be pulled out into separate classes.